generator/internal/rust/templates/crate/src/transport.rs.mustache

```diff
@@ -103,14 +99,14 @@ impl super::stub::{{Codec.Name}} for {{Codec.Name}} {
                 {{/Codec.QueryParams}}
                 Ok(builder)
             })();
-            Some(builder)
             {{/Codec.QueryParamsCanFail}}
             {{^Codec.QueryParamsCanFail}}
             {{#Codec.QueryParams}}
             {{{Codec.AddQueryParameter}}}
             {{/Codec.QueryParams}}
-            Some(Ok(builder))
+            let builder = Ok(builder);
```

Comment:
The output from this is quite odd. It may be better to duplicate some code in the mustache template to produce more natural code. 🤷